src/cartridges/bm_adyen/cartridge/static/default/js/adyenSettings.js

```diff
@@ -465,6 +468,14 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   });
 
+  productionRadio.addEventListener('click', () => {
+    livePrefix.disabled = productionRadio.value === 'TEST';
```

Comment:
this should be 
`livePrefix.disabled = false` 
right now it works because productionRadio.value will always be LIVE ;)